source/platforms/github/GitHubAPI.ts

```diff
@@ -223,6 +268,17 @@ export class GitHubAPI {
     return await this.getAllOfResource(`repos/${repo}/issues/${prID}/comments`)
   }
 
+  getPullRequestInlineComments = async (): Promise<Comment[]> => {
+    const userID = await this.getUserID()
+    const repo = this.repoMetadata.repoSlug
+    const prID = this.repoMetadata.pullRequestID
+    return await this.getAllOfResource(`repos/${repo}/pulls/${prID}/comments`).then(v => {
+      return v.map((i: any) => {
+        return { id: i.id, ownedByDanger: i.user.id == userID, body: i.body }
```

Comment:
I think this will want to also have a danger-id check (could be reasonable to imagine many dangers posting inline)